packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -6,14 +6,18 @@
 import os
 import random
 import sys
-
+from subprocess import check_call, CalledProcessError
 
 from dcos_internal_utils import bootstrap
 from dcos_internal_utils import exhibitor
 
+from pkgpanda import load_json
 
 log = logging.getLogger(__name__)
 
+_DCOS_SERVICE_CONFIGURATION_JSON = "/opt/mesosphere/dcos-service-configuration.json"
```

Comment:
nothing to be done here. Yeah, I agree that constants can factored out from other places too.